SPECS/prometheus/prometheus.signatures.json

```
@@ -1,11 +1,11 @@
 {
  "Signatures": {
-  "prometheus-2.37.0.tar.gz": "98892e82b97004a458e81f03d804859d485323af2d85c34f8a996e25fe1305a9",
+  "prometheus-2.37.9.tar.gz": "f26eba405e0836c5a53bfff91b45dc71b14900d5edc0fe8db7238d3c85ac45fb",
   "prometheus.conf": "ce522e82dfb2945c520b482b15b5cf591364f7a571f0f28259b64dbeda42b043",
   "prometheus.logrotate": "061b92500cd40fcaaf486ff488bcf1b09eac6743d8e840ba6966dc70d4e2067b",
   "prometheus.service": "29bf1c886e1d55080e859f2afe112bb7344490e6992e946efe3360fd94d1a604",
   "prometheus.sysconfig": "ec89a45641e3411478794106246aa91e7b72f86070a28a4782e3b8be955e4587",
   "prometheus.yml": "0112e0bf54660c5e2391fff11a56404a25684c588caa7281677f7f8e19da6f28",
-  "promu-0.13.0.tar.gz": "3473b87214968c79158f553228baef6e9a37ed3e11e1a4f3e7267ffd3180a8b6"
+  "promu-0.14.0.tar.gz": "1e07b4a6d7f00cb96b3d4c4d8ce3d463501cb0a0f790aed32a87985ce63dab5d"
```

Looks like there's a bug in the publishing pipeline. This file was already in our blobstore but with a different hash. We've restored it to fix the 3.0 builds, which started failing after the upload of this one. Please use this hash instead and it should unblock you as well:
```suggestion
  "promu-0.14.0.tar.gz": "d71d2a0d54093f3f17dc406d7a5825b6d6acd304cd90d9c60ed3f1335fb6ed2a"
```

---

Also, please follow [these steps to upload vendored tarballs](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/1284/Source-Tar-Publishing?anchor=upload-vendored-source-tarball%3A), so we don't create new blobstores or other file servers, which may be hard to track or secure.

---

Thanks @PawelWMS. Sure.